### PR TITLE
Fix styling issues

### PR DESCRIPTION
**Pull request by Matthijsy**
Time to re-enable some of the eslint rules and remove some anti-patterns!

### CODE REVIEW

File: app/templates/components/form/responses-table.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
     {{/if}}
     <th> Naam </th>
     {{#each form.sortedQuestions as |question|}}
-      <th> {{question.question}} </th>
+      <th> {{question.question}} - {{question.position}} </th>
```
Comment (guidojw) [inline app/templates/components/form/responses-table.hbs:8]: This starts with 0

Maybe a nicer way of doing this is showing it in the Inschrijven form as "{{pos}}. {{question}}" and then have that format here as well?

**Review by guidojw — CHANGES_REQUESTED** [review]
Apart from the comments mentioned I found a bug with opening activities:
The first time opening any activity from the authenticated home page it shows the show page correctly, but when I then go back and open the page again, I get the following error:

Screenshot_1551

File: app/templates/forum/categories/category/threads/thread/posts/index.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 {{#each @model.posts as |post| }}
   {{#unless post.isDeleted}}
-    {{forum/forum-post post newContent=newContent quoteAction=(action 'goToLastPageAndScrollDown') data-test-forum-post=post.id}}
```
[the remark this answers is not in the document]
Comment (Matthijsy) [reply app/templates/forum/categories/category/threads/thread/posts/index.hbs:11]: This actually is already broken before this PR, we have to fix that but then in a way that doesn't use closure actions (how this is called) anymore. I propose to create a bug issue for it and fix it in a later PR

File: app/templates/components/form/responses-table.hbs (hunk 1 above)
Comment (Matthijsy) [reply app/templates/components/form/responses-table.hbs:8]: No it shouldn't show the positon, that was for debug, will remove it

File: app/components/md-editor.js [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
 
-  @computed('elementId')
-  get textareaId() {
-    return `${this.elementId}-textarea`;
-  }
+  content = '';
+  @tracked textareaId;
 
   @action
   applyStyle(option, modalInput = null) {
```
[the remark this answers is not in the document]
Comment (guidojw) [reply app/components/md-editor.js:13]: Style is now getting applied correctly but I can't submit forms with an md-editor anymore

Screenshot_1552

[the remark this answers is not in the document]
Comment (Matthijsy) [reply app/components/md-editor.js:13]: This is an issue from using the new style objects combined with the legacy once, so I think we can fix this after we migrated all the components and the controllers. For now I have left the bug where the styling doesn't work, are you okay with fixing this after the full migration?

[the remark this answers is not in the document]
Comment (guidojw) [reply app/components/md-editor.js:13]: Okay sounds good!

**Review by guidojw — APPROVED** [review]

**Pull request merged**